src/util/accessibility/keyboardNavigation.ts

```diff
@@ -6,7 +6,10 @@ import KeyboardNavigationEventListener, {
 
 export { NavigationMode, NavigationModeCallback };
 
-const navListener = new KeyboardNavigationEventListener(document.body);
+let navListener: KeyboardNavigationEventListener;
```

Comment:
Now the true type of this variable is `KeyboardNavigationEventListener | undefined`.

Reply:
(because until `DOMContentLoad` it will be undefined)

Reply:
So you'll need to add `if` guards wherever we access the variable. Otherwise shit would blow up if the method gets called before `DOMContentLoad`.